transformation/src/api/amqp/pipelineConfigConsumer.ts

```diff
@@ -0,0 +1,83 @@
+import * as AMQP from 'amqplib'
+import { PipelineConfigManager } from '../../pipeline-config/pipelineConfigManager'
+import pipelineConfigTriggerRequest from '../pipelineConfigTriggerRequest'
```

Comment:
Use `env.ts` without default values for fail-fast on startup with wrong configuration